src/main/java/frc/team2412/robot/util/AutonomousChooser.java

```diff
@@ -83,7 +83,7 @@ private static SequentialCommandGroup getAutoWPICommand(Subsystems subsystems) {
         STAR_PATH((subsystems, trajectories) -> AutonomousChooser.getStarPathAutoCommand(subsystems, trajectories), "Star Path"),
         WPI_PATH((subsystems, trajectories) -> AutonomousChooser.getAutoWPICommand(subsystems), "WPI Lib Path"),
         CLIMB((subsystems, trajectories) -> new ClimbTestCommand(subsystems.climbSubsystem), "Climb test"),
-        INDEX((subsystems, trajectories) -> new IndexTestCommand(subsystems.indexSubsystem), "Index test"),
+        INDEX((subsystems, trajectories) -> new IntakeInCommand(subsystems.indexSubsystem, subsystems.intakeSubsystem), "Index test"),
```

Comment:
Does use of this command put a dependency on intake so we can't run diagnostic tests on index without intake being on the robot and enabled?

Reply:
Yes, `IntakeInCommand` requires index and intake.
`IntakeInCommand` is a [`SequentialCommandGroup`](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/SequentialCommandGroup.html) composed of an `IntakeExtendCommand`, a `WaitCommand`, and a [`ParallelCommandGroup`](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/SequentialCommandGroup.html) composed of an `IntakeMotorInCommand` and an `IndexShooterCommand`. The docs for sequential and parallel command groups say, "As a rule, CommandGroups require the union of the requirements of their component commands.", and since `IntakeExtendCommand`, `IntakeMotorInCommand`, and `IndexShooterCommand` require their respective subsystems, the `IntakeInCommand` as a whole requires both index and intake.